client-ts/signalr/src/HttpConnection.ts

```diff
@@ -68,9 +68,9 @@ export class HttpConnection implements IConnection {
             }
             else {
                 let headers;
-                if (this.options.accessToken) {
+                if (this.options.accessTokenFactory) {
```

Comment:
Curious: is `accessTokenFactory` allowed to return a null/empty token? Should a non-null token check be added?

Reply:
I guess it could, and I don't see why we couldn't "allow" it (and just not provide the header).